posts/2025-10-19-why-study-aristotle-categories.md

```
@@ -0,0 +1,85 @@
+## Plato System: the Realm of Forms
+
+Before Aristotle, Plato already devised a system that can be used to answer those questions. However, Plato's system demands a new reality: **the realm of Forms**.
+
+When Socrates asked “What is Justice?”, Plato said:
+
+> Justice itself — Justice with a capital J — is not found in the world around us, but in a higher, unchanging realm of Forms (or Ideas).
+
+So, in Plato’s view:
+
+-   Every thing in the physical world “participates in” or “imitates” its **Form**.
+    -   Example: All beautiful things share in **the Form of Beauty**.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)